Sources/ConcentricOnboarding/ConcentricOnboardingView.swift

```diff
@@ -14,9 +14,9 @@ class ObservableBool: ObservableObject {
             didSet?()
         }
     }
-
+    
```

Comment:
There shouldn't be spaces in empty lines (I believe it's some sort of xcode setting), could you please restore my completely empty lines?

Reply:
I have restored it back.... and this time around the extra spaces are gone.... please check it out and let me know...